core/src/main/resources/xml/xsl/esb/soapBody.xsl

```diff
@@ -141,22 +155,62 @@
 									</xsl:element>
 								</xsl:element>
 							</xsl:element>
-							<xsl:if test="string-length($errorDetailCode)&gt;0">
-								<xsl:element name="DetailList" namespace="{$ns}">
-									<xsl:element name="Detail" namespace="{$ns}">
-										<xsl:element name="Code" namespace="{$ns}">
-											<xsl:value-of select="$errorDetailCode"/>
-										</xsl:element>
-										<xsl:if test="string-length($errorDetailText)&gt;0">
+							<xsl:choose>
+								<xsl:when test="$addErrorsFromInputMessage='true' and string-length($errorRootTag)&gt;0 and $rootTag='errorMessage'">
+									<xsl:element name="DetailList" namespace="{$ns}">
+										<xsl:element name="Detail" namespace="{$ns}">
+											<xsl:element name="Code" namespace="{$ns}">
+												<xsl:text>ERROR_1</xsl:text>
```

Comment:
Ik zie dat je ERROR_1, ERROR_2 e.d. als code gebruikt. Ik denk dat het handiger is om in die gevallen alleen 'ERROR' te gebruiken, omdat je dan xpath queries kan doen als /DetailList/Detail[Code='ERROR']/Text.

Reply:
Aangepast zoals door jou voorgesteld